CMakeLists.txt

```diff
@@ -99,6 +99,8 @@ if(GFX_ENABLE_GUI)
     target_compile_definitions(gfx PUBLIC GFX_ENABLE_GUI)
 endif()
 if(GFX_ENABLE_SCENE)
+    option(GFX_ENABLE_SCENE_KTX "Build gfx with ktx image loading support" ON)
```

Comment:
Also added an option so that ktx support is optional as the ktx libs are rather heavy and with dds they may not be wanted (kept them default enabled for backwards compat)